source/mongorestore.txt

```diff
@@ -1234,6 +1243,71 @@ the filename. For example:
 
    mongodump --archive --db=test --port=27017 | mongorestore --archive --port=27018
 
+
+.. _mongorestore-example-time-series:
+
+Restore a Time Series Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use ``mongosh`` to create a time series collection. This example 
+uses the default ``test`` database:
+
+.. code-block:: javascript
+
+   db.createCollection(
+      "weather",
+      {
+         timeseries: {
+            timeField: "timestamp",
+            metaField: "metadata",
+            granularity: "hours"
+         }
+      }
+   )
+
+Insert time series documents into the collection:
+
+.. code-block:: javascript
+
+   db.weather.insertMany( [
+      {
+         "metadata": { "sensorId": 5578, "type": "temperature" },
+         "timestamp": ISODate("2021-05-18T00:00:00.000Z"),
+         "temp": 12
+      },
+      {
+         "metadata": { "sensorId": 5578, "type": "temperature" },
+         "timestamp": ISODate("2021-05-18T04:00:00.000Z"),
+         "temp": 11
+      },
+      {
+         "metadata": { "sensorId": 5578, "type": "temperature" },
+         "timestamp": ISODate("2021-05-18T08:00:00.000Z"),
+         "temp": 11
+      }
+   ] )
+
+From your terminal, use ``mongodump`` to export the time series collection 
+to the ``dump/test`` directory. This command adds two files 
```

Comment:
[nit]

Consider one of the following:

- "This command adds two files to the directory: \``system.buckets.weather.bson\`` and \``weather.metadata.json``.
- "This commands adds \``system.buckets.weather.bson\`` and \``weather.metadata.json\`` to the directory.